tests/prefetch_related/tests.py

```diff
@@ -1161,3 +1161,84 @@ def test_bug(self):
         prefetcher = get_prefetcher(self.rooms[0], 'house')[0]
         queryset = prefetcher.get_prefetch_queryset(list(Room.objects.all()))[0]
         self.assertNotIn(' JOIN ', force_text(queryset.query))
+
+
+class UUIDPrefetchRelated(TestCase):
+
+    def test_prefetch_related_from_uuid_model(self):
+        Pet.objects.create(name='Fifi').people.add(
+            Person.objects.create(name='Ellen'),
+            Person.objects.create(name='George')
```

Comment:
In cases like this, we prefer to include a trailing comma so if more items are later added, we don't need to modify this line again.